blogo-input/posts/votedelegation_thumb.md

```diff
@@ -0,0 +1,3 @@
+### Adding Vote Delegation to Anonymous E-Voting Schemes
+
```

Comment:
@vincenzoiovino you can add here a one-line description of the post, which will be listed in the articles list of the main page